src/Mission.scss

```diff
@@ -0,0 +1,16 @@
+.map{
+    width:100%;
+    background-image: url("/images/mars-map.jpg");
+    background-repeat:no-repeat;
+    background-size:100%;
+}
+
+.circle {
+    fill: red;
```

Comment:
Could you add this colour and the cyan colour to `constants.scss`?
That way we can reuse the same colour palette throughout the site